src/openslide-decode-tiff.c

```diff
@@ -683,3 +683,57 @@ void _openslide_tiffcache_destroy(struct _openslide_tiffcache *tc) {
   g_free(tc->filename);
   g_free(tc);
 }
+
+bool _openslide_tiff_icc_profile_size(openslide_t *osr G_GNUC_UNUSED,
```

Comment:
As discussed above, this function can be dropped.  We'll get the size at open time via `tifflike`.